.buildkite/container.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh
 
+set -eu
+
+image_tag=$1
```

Comment:
@finanalyst the container build script that CI uses now takes a parameter. Merges to main will still push latest, which was previously hardcoded.